tests/editor/utils/dataapimixin.js

```diff
@@ -22,10 +22,11 @@ describe( 'DataApiMixin', () => {
 		editor.model.document.createRoot( '$root', 'main' );
 		editor.model.document.createRoot( '$root', 'secondRoot' );
 		editor.model.schema.extend( '$text', { allowIn: '$root' } );
+		editor.fire( 'ready' );
 	} );
 
-	afterEach( () => {
-		editor.destroy();
+	afterEach( async () => {
```

Comment:
Nit: I see you really love `async` / `await` structure :smiley: Since `editor.destroy()` returns a promise, a shorter way would be to simply `afterEach( () => editor.destroy() );` but it's just a matter of preference 😉